pkg/operator/workloadcontroller/workload_controller.go

```diff
@@ -276,3 +288,74 @@ func (c *ServiceCatalogAPIServerOperator) namespaceEventHandler() cache.Resource
 		},
 	}
 }
+
+func (c ServiceCatalogAPIServerOperator) deleteOwnerRefFromServiceBindings() error {
+	// Get service bindings
+	gvr := schema.GroupVersionResource{
+		Group:    "servicecatalog.k8s.io",
+		Version:  "v1beta1",
+		Resource: "servicebindings"} // lowercase plural is important
+
+	// get all of the service bindings
+	bindings, err := c.dynamicClient.Resource(gvr).List(metav1.ListOptions{})
+	if err != nil {
+		if apierrors.IsNotFound(err) {
+			klog.Info("No service bindings found, nothing to delete.")
+			return nil
+		}
+		klog.Errorf("Problem getting the bindings list. %v", err)
+		return err
+	}
+
+	klog.Infof("we have %d binding items", len(bindings.Items))
+	for _, binding := range bindings.Items {
+		// get the secret name
+		secretName, exists, err := unstructured.NestedString(binding.Object, "spec", "secretName")
+		if err != nil {
+			klog.Errorf("binding %s has a secretName field that is not a string: %v", binding.GetName(), err)
+			return err
+		}
+		if !exists {
+			klog.Warningf("binding exists without a secretName! Nothing to delete.")
+		}
+
+		// get the namespace
+		namespace, exists, err := unstructured.NestedString(binding.Object, "metadata", "namespace")
+		if err != nil {
+			klog.Errorf("binding %s has a namespace field that is not a string: %v", binding.GetName(), err)
+			return err
+		}
+		if !exists {
+			klog.Warningf("binding exists without a namespace! Nothing to delete.")
+		}
+
+		klog.Infof("our binding has a secret %s in namespace %s", secretName, namespace)
+
+		// Get the service binding's secret
+		secret, err := c.kubeClient.CoreV1().Secrets(namespace).Get(secretName, metav1.GetOptions{})
+		if err != nil && apierrors.IsNotFound(err) {
+			// if we can't find the secret for the service binding, continue
+			klog.Warningf("binding %s does not have secret %s not found, skipping", binding.GetName(), secretName)
```

Comment:
Nit: wording is a little confusing for this log message.

```suggestion
			klog.Warningf("secret %s for binding %s not found, skipping", secretName, binding.GetName())
```